lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -46,7 +46,9 @@
 
         $user = user_load($task->drupal_id);
 
-        dosomething_rogue_send_reportback_to_rogue($values, $user);
+        $rogue_reportback = dosomething_rogue_send_reportback_to_rogue($values, $user);
+
+        dosomething_rogue_check_rbid_and_store_ref($rogue_reportback);
```

Comment:
@angaither hmm good question. I guess in this particular case we don't care what it returns because `dosomething_rogue_send_reportback_to_rogue` takes care of stuff ending up in the failed table. But elsewhere we want to know if it is false so we can hit the permalink failure page.